File: Hypothesis.R
getwd()
setwd("C:/Users/chris/Desktop/20BCAB23")
car=read.csv("CarPrice_Assignment.csv")
mean(car$horsepower)
# H0: Mean Horsepower = 104.1171
# H1: Mean Horsepower > 104.1171
t.test(car$horsepower,mu=104.1171,alternative ='two.sided',conf.level=0.95)

File: Hypothesis.py
# -*- coding: utf-8 -*-
"""
Created on Thu Mar 24 22:39:36 2022

@author: fiona
"""

getwd()
data=read.csv("CarPrice_Assignment.csv")
mean(data$curbweight)
# H0: Mean curbweight = 2550
# H1: Mean curbweight > 2550
t.test(data$curbweight,mu=2550,alternative ='two.sided',conf.level=0.95)